src/main/java/com/fundation/webservice/model/CriteriaAudio.java

```diff
@@ -51,7 +51,7 @@ public String getAudioCodec() {
 	/**
```

Comment:
Align these comments.

Reply:
There's nothing to align. This is the standard javadoc layout.